### DISCUSSION TITLE
Firefox's issue with module import inside Web Worker

### DISCUSSION THREAD

**Discussion by DirectorRen-TV** [nuxt/nuxt#20671] [Questions]
Currently (version 112) for some reasons this feature is hidden under `about:config` → `dom.workers.modules.enabled` that is `false` value by default. 
Accordingly, it is necessary to avoid them. But Nuxt imports .env.mjs into all modules, and although the documentation says `Nuxt has built-in support for loading .env files. Avoid directly importing it from nuxt.config.` there is no setting to disable automatic import for .env file. Turning off all auto-import makes import including `defineNuxtConfig`, but from `#imports` or `nuxt` it does not want to import.
Is there any way to disallow auto-importing .env into at least one single file?
Error:
![изображение](https://user-images.githubusercontent.com/39501885/236188207-70970195-dcd1-4651-a086-899814570e4a.png)

source code from browser
![изображение](https://user-images.githubusercontent.com/39501885/236188365-f7a413f2-8ba1-4981-9967-4f69decb66aa.png)

worker creation
![изображение](https://user-images.githubusercontent.com/39501885/236188630-31b9e13c-24ef-46a1-bf03-36c358c63d43.png)
or (works similar in this case)
![изображение](https://user-images.githubusercontent.com/39501885/236188741-1ade2639-9f9c-4a21-b6ed-7f8ddb2c4ecd.png)

**Comment by logotip4ik** [comment]
https://github.com/mdn/content/issues/24402#issuecomment-1519226816

**Reply by DirectorRen-TV** [reply]
The news is good and in principle I can already enable support through config, but in general it should take some time, first before the release of the 113 update, and then also until the majority of users will update.
I think the topic touched is also about the framework itself - if the documentation says it can be done, then it must be possible to do it in the way the documentation says.
